packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,24 @@
+This tutorial will be exploring different ways of rendering a list of items in
+Lit!
+
+Lit makes it easy to render any lists of items. It has built-it support for
+iterables of any supported values. This, along with Lit's ability to render
+nested templates and helpful built-in directives, allow for rendering any
+variety of lists.
+
+## What you'll learn
+- Utilizing the `map` directive
+- Using array prototype methods
+- Building static list and using the `range` directive
```

Comment:
The term "static list" might not be immediately apparent. Maybe "Generating lists with the range directive"